src/Plugin/QueueWorker/HubspotCommerceQueue.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Drupal\hubspot_commerce\Plugin\QueueWorker;
+
+use Drupal\hubspot_commerce\Services\HubspotCommerce;
+
+use Drupal\Core\Annotation\QueueWorker;
+use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Class HubspotCommerceQueue.
+ *
+ * @QueueWorker(
+ *   id = "hubspot_commerce_queue",
+ *   title = @Translation("Hubspot Commerce queue"),
+ *   cron = {"time" = 60}
+ * )
+ */
+class HubspotCommerceQueue extends QueueWorker implements ContainerFactoryPluginInterface {
```

Comment:
**CHANGE REQUEST**

Have you tested this? Where did you copy the code from? You should be extending the `QueueWorkerBase` instead of the class that defines the annotations for QueueWorker plugins.

The class should be renamed to `EntitySyncTo`. We need the prefix in the plugin's ID for namespacing (avoid collisions with other modules) but it is not necessary in the class itself as it is namespaced already. We don't need a suffix; that depends on conventions, for some type of classes the convention is to use a suffix indicating what is this class, like for event subscribers e.g. `OrderNumberSubscriber` but if you look at core or contrib modules the convention for queue workers is to not have a suffix.